packages/vite-plugin/src/index.ts

```diff
@@ -56,15 +58,17 @@ export function vanillaExtractPlugin({ identifiers }: Options = {}): Plugin {
       packageInfo = getPackageInfo(config.root);
     },
     resolveId(id) {
-      if (id.indexOf(virtualExt) > 0) {
+      if (id.startsWith(virtualPrefix) && id.indexOf(virtualExt) > 0) {
         return id;
       }
     },
     load(id) {
       const extensionIndex = id.indexOf(virtualExt);
 
-      if (extensionIndex > 0) {
-        const fileScopeId = id.substring(0, extensionIndex);
+      if (id.startsWith(virtualPrefix) && extensionIndex > 0) {
+        const fileScopeId = id
+          .substring(0, extensionIndex)
+          .slice(virtualPrefix.length);
```

Comment:
This can be done through one call to `slice`. e.g. `slice(virtualPrefix.length, extensionIndex)`

Reply:
Very good point. Done. Thank you